KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -624,6 +624,8 @@
 
 "Text.Quote.FreeCancellationBeforeDriverEnRoute" = "Free cancellation until the driver is en route";
 
+"Text.Quote.FeesAndTaxesIncluded" = "Taxes and fees included";
```

Comment:
You need to add the new key/value to Phrase and translate them for the other languages as well